package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-red-contrib-df1",
-  "version": "1.1.2",
+  "version": "1.2.0",

```

Comment:
Are we adding a new feature on this PR? I believe only the patch version should be incremented here